tutorials/flarefly_tutorial_Bzero.ipynb

```diff
@@ -0,0 +1,399 @@
+        "import matplotlib.pyplot as plt\n",
+        "import uproot\n",
+        "from flarefly.data_handler import DataHandler\n",
+        "from flarefly.fitter import F2MassFitter"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "1o8a7oPVnxE3"
+      },
+      "source": [
+        "## **Prepare data | DataHandler instance**\n",
+        "\n",
+        "The data one aims at fitting are drawn from the histogram *hist_mass_bzero_pt0_50* contained in *file.root* downloaded earlier.\n",
+        "\n",
+        "*flarefly* allows for a simple handling of the data by calling a *DataHandler* instance that shall take care of adapting the input to our fitter class."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "1me0iMExNRDA"
+      },
+      "outputs": [],
+      "source": [
+        "data = DataHandler(data=\"file.root\",\n",
+        "                   var_name=r\"$M_\\mathrm{D^{-}\\pi^{+}}$ (GeV/$c^{2}$)\",\n",
+        "                   histoname=\"hist_mass_bzero_pt0_50\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "p5CvkIgsPPlS"
+      },
+      "source": [
+        "## **Prepare fitter | F2MassFitter instance**\n",
+        "\n",
+        "Once data are prepared, one still needs to configure the fitter:\n",
+        "\n",
+        "1. Define probability density functions (PDFs) for signal and background\n",
+        "2. Define a F2MassFitter instance\n",
+        "3. Initialize fit parameters"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "z3HLaZUNTntt"
+      },
+      "source": [
+        "### **1. Define PDFs**\n",
+        "\n",
+        "*flarefly* is based on *zfit*, itself optimised for a direct manipulation of probability density functions. Hence the need to define PDFs for signal and for background.\n",
+        "\n",
+        "**NB**: *flarefly* allows one to perform multi-fits with several signal and/or background PDFs. That's why one works with PDFs **lists**. In this sense, every PDF is associated to an index (i.e. its position in the list). This index will be required later while configuring the fitter."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "aSxK1wUpP3Vz"
+      },
+      "outputs": [],
+      "source": [
+        "# define PDFs lists\n",
+        "signal_pdfs = [\"gaussian\"]\n",
+        "background_pdfs = [\"expo\"]\n",
+        "\n",
+        "# define the ids\n",
+        "gaussian_id = 0 # because signal_pdfs[gaussian_id] = \"gaussian\"\n",
+        "expo_id = 0     # because background_pdfs[expo_id] = \"expo\""
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rrXKVQgHVBPF"
+      },
+      "source": [
+        "### **2. Define a F2MassFitter instance**\n",
```

Comment:
Maybe it is worth mentioning that the name of the F2MassFitter has to be unique for each fitter one wants to use